src/core/lib/iomgr/event_engine/uv/impl.cc

```diff
@@ -286,12 +209,16 @@ absl::StatusOr<int> uvListener::Bind(
     int r = uv_tcp_bind(&uvTCP_->tcp_, addr.address(), 0 /* flags */);
     switch (r) {
       case UV_EINVAL:
-        p.set_value(-1 /*util::Status(util::error::INVALID_ARGUMENT, "uv_tcp_bind said we passed an invalid argument to it")*/);
+        p.set_value(absl::Status(
```

Comment:
This can be shortened to `p.set_value(absl::InvalidArgumentError("uv_tcp_bind said ..."));`

https://github.com/abseil/abseil-cpp/blob/master/absl/status/status.h#L722-L740

Same below